### PR TITLE
docs: improve configuring supply chains with Synk integration

### PR DESCRIPTION
**Pull request by rawlingsj**
Which other branches should this be merged with (if any)?

### CODE REVIEW

**Comment by rawlingsj** [comment]
Please hold for review by scan team

File: scst-scan/install-snyk-integration.md [hunk 1]
```diff
@@ -133,15 +133,15 @@ To install Supply Chain Security Tools - Scan (Snyk scanner):
      Added installed package 'snyk-scanner' in namespace 'tap-install'
     ```
 
-## Using the Synk scanner to run an ImageScan
+## Verify integration with Synk 
 
-The out of the box Scanning Supply Chain implements a `SourceScan` and `ImageScan`, which reference `ScanTemplate`s that use the Grype scanner. To use the Snyk scanner for the `ImageScan`, the `ImageScan` `ScanTemplate` must be used instead. In the out of the box Scanning Supply Chain, update `spec.scanTemplate` from `public-image-scan-template` to `snyk-public-image-scan-template`.
+To verify the integreation with Synk you can apply the following `ImageScan` in the Developer namespace and check the result.
```
Comment (xtreme-conor-nosal) [inline scst-scan/install-snyk-integration.md:138]: typo in integration

**Comment by xtreme-conor-nosal** [comment]
Looks good

File: scst-scan/install-snyk-integration.md (hunk 1 above)
Comment (rawlingsj) [reply scst-scan/install-snyk-integration.md]: good catch! thanks

**Comment by rawlingsj** [comment]
thanks - taken off draft.

**Comment by snneji** [comment]
Thanks @rawlingsj

**Pull request merged**